scripts/index.js

```diff
@@ -0,0 +1,108 @@
+// TEMP  
+const changeTemp = (temp) => {
+    let tempColor = "yellow"
+    let garden = "👙🌻🌞🩴🏖👙🌻🌞🩴🏖"
+    if (temp > 85) {
+        tempColor = "red";
+        garden = "👙🌻🌞🩴🏖👙🌻🌞🩴🏖";
+    } else if (temp > 75) {
+        tempColor = "orange";
+        garden = "⛲️⛵️🌤🌺🌳⛲️⛵️🌤🌺🌳"; 
+    } else if (temp > 60){
+        tempColor = "yellow";
+        garden = "🌳🌲🌷☁️💨🌳🌲🌷☁️💨";
+    } else if (temp > 50){
+        tempColor = "green";
+        garden = "☁️🌫🍂🌲🍃☁️🌫🍂🌲🍃";
+    } else if (temp < 50){
+        tempColor = "blue";
+        garden = "🌬🍁❄️☃️🧥🌬🍁❄️☃️🧥";
+    }
+    
+    const newGarden = document.querySelector("#garden-landscape");
+    newGarden.textContent = garden
+    return tempColor
+}
+
+const increaseTemp = () => {
+    const newTemp = document.querySelector("#temp");
+    const newTempValue = parseFloat(newTemp.textContent) + 1;
+    tempColor = changeTemp(newTempValue);
+    newTemp.className = tempColor;
+    newTemp.textContent = String(newTempValue);
+};
+
+const decreaseTemp = () => {
+    const newTemp = document.querySelector("#temp");
+    const newTempValue = parseFloat(newTemp.textContent) - 1;
+    tempColor = changeTemp(newTempValue);
+    newTemp.className = tempColor;
+    newTemp.textContent = String(newTempValue);
+    
+};
+
+
+// SKY 
+const skyView = (option) => {
+    let currentSky =  "☀️☀️☀️☀️☀️☀️"
+    if (option == "sunny") {
+        currentSky = "☀️☀️☀️☀️☀️☀️";
+    } else if (option=="cloudy") {
+        currentSky = "☁️☁️☁️☁️☁️";
+    } else if (option=="rainy") {
+        currentSky = "🌧🌧🌧🌧🌧";
+    } else if (option=="snowy") {
+        currentSky = "🌨❄️🌨❄️🌨❄️";
+    }
+    const changeSky = document.querySelector("#garden-sky");
+    changeSky.textContent = currentSky
+}
+
+const selectSky = () => {
+    const sky_selection = document.querySelector("#sky-dropdown")
+    skyView(sky_selection.value)
+}
+
+
+// CITY 
+const changeCity = (event) => {
+    let inputCity = document.querySelector("#city-name").value;
+    let mainHeading = document.getElementById("default-city");
+    if (!inputCity == "") {
+        mainHeading.innerHTML = `Weather Report for: ${inputCity}`;
+        inputCity.value = "";
```

Comment:
inputCity is the value of the textbox, not the textbox itself.  setting inputCity.value will attach a `value` property to the string (which doesn't quite work, but JavaScript won't throw an error).  To reset the textbox, something like `document.querySelector("#city-name").value = ""` is needed.